docs/conceptual/core/codec.md

```diff
@@ -1,13 +1,146 @@
 ---
-title: Parity Scale Codec
+title: SCALE Codec
 ---
 
-* What is parity scale codec?
+The SCALE (Simple Concatenated Aggregate Little-Endian) Codec is a lightweight, efficient, binary serialization and deserialization codec.
 
-* How does it work?
+It is designed for high-performance, copy-free encoding and decoding of data in resource-constrained execution contexts, like the [Substrate runtime](conceptual/runtime/index.md). It is not self-describing in any way and assumes the decoding context has all type knowledge about the encoded data.
 
-* Examples
+## SCALE for Substrate
 
-* Why did we invent our own codec?
+Substrate uses the [`parity-scale-codec`](https://github.com/paritytech/parity-scale-codec), a Rust implementation of the SCALE Codec. This library and the SCALE codec is advantageous for Substrate and blockchain systems because:
 
-* What implementations exist?
\ No newline at end of file
+* It does not use Rust STD, and thus can compile to Wasm.
+* It is built to have great support in Rust for deriving codec logic for new types: just add `#[derive(Encode, Decode)]`.
+
+It is important to define the encoding scheme used on Substrate rather than reuse an existing Rust codec library because this codec needs to be re-implemented on other platforms and languages that want to support interoperability.
```

Comment:
I don't really understand this claim. Isn't reuse on other platforms and in other languages a reason _to_ use an already-popular codec? I thought parity invented a new one to be super lightweight. Don't remember why u thought that though.

Reply:
I think the thing I am trying to get across is that _we_ control the standard used here. Basti was telling me that there are probably already existing codecs which are as light weight as this, and implemented in rust.

However, those codecs may not have a well defined specification (so that they can be implemented in other languages), and may change or break in some way outside of our control.

We define the spec in the Polkadot Runtime Environment specification, and thus we control how the codec evolves over time if needed.

Does that make sense? Should I just say this instead?